### PR TITLE
added a knn+posenet example, update knn+video example

### PR DESCRIPTION
**Pull request by yining1023**
This is a knn+posenet example that classifies poses from webcam.

Edit: This PR also update the knn+video example to use the updated API functions.

### CODE REVIEW

File: p5js/KNNClassification/KNNClassification_PoseNet/sketch.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+  select('#status').html('model Loaded')
+}
+
+// Add the current frame from the video to the classifier
+function addExample(label) {
+  // Convert poses results to a 2d array [[score0, x0, y0],...,[score16, x16, y16]]
+  const poseArray = poses[0].pose.keypoints.map(p => [p.score, p.position.x, p.position.y]);
+
+  // Create a tensor2d from 2d array
+  const logits = ml5.tf.tensor2d(poseArray);
```
[the remark this answers is not in the document]
Comment (yining1023) [reply p5js/KNNClassification/KNNClassification_PoseNet/sketch.js]: Yes! I agree. Will open a PR on ml5-library.

File: p5js/KNNClassification/KNNClassification_PoseNet/sketch.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+// Predict the current frame.
+function classify() {
+  // Get the total number of labels from knnClassifier
+  const numLabels = knnClassifier.getNumLabels();
+  if (numLabels <= 0) {
+    console.error('There is no examples in any label');
+    return;
+  }
+  // Convert poses results to a 2d array [[score0, x0, y0],...,[score16, x16, y16]]
+  const poseArray = poses[0].pose.keypoints.map(p => [p.score, p.position.x, p.position.y]);
```
Comment (shiffman) [inline p5js/KNNClassification/KNNClassification_PoseNet/sketch.js]: Noting that this is something we may want to simplify later as discussed in https://github.com/ml5js/ml5-library/issues/245 and https://github.com/ml5js/ml5-library/issues/255. It's also an interesting question as to whether or not to include `p.score` and/or what to do with keypoints that have a very low confidence score.

**Comment by cvalenzuela** [comment]
Looks great!

**Comment by cvalenzuela** [comment]
I think we are good to merge this. https://github.com/ml5js/ml5-library/pull/256 seems ready too

**Pull request merged**